src/components/TextField/TextField.tsx

```diff
@@ -45,12 +51,18 @@ export const TextField: React.FC<Props> = ({
           })}
           placeholder={placeholder}
           value={value}
-          onChange={event => onChange(event.target.value)}
+          onChange={event => onChange(event)}
```

Comment:
You can try to write just 
```suggestion
          onChange={onChange}
```